### ISSUE TITLE
run yelp mode sample 0.05% 3 runs de 10 epochs  - é muito pequeno 

### ISSUE THREAD

**Issue by IsabelCanicoNeto** [jmribeiro/dsl-project#31]
C:\ProgramData\Anaconda3\python.exe "C:/Users/isabe/DL 2019 HW/dsl-project/run_dataset.py" yelp -sample -dataset_size 0.0005 -epoch 10 -nrun 3 -tqdm
.data\yelp_review_full_csv.tar.gz: 196MB [00:14, 13.5MB/s]
650000lines [01:39, 6556.77lines/s]
650000lines [02:42, 3992.12lines/s]
50000lines [00:10, 4849.62lines/s]
==== Split train.csv and test.csv files =====
.data/yelp_review_full_csv/train
Delete /train.csv
Rename /train_sample.csv to /train.csv
.data/yelp_review_full_csv/test
Delete /test.csv
Rename /test_sample.csv to /test.csv
 (Done) [292 training samples]
*** run number  1 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
 (Done)
  0%|          | 0/5 [00:00<?, ?it/s]*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
 80%|████████  | 4/5 [01:48<00:27, 27.01s/it]Training loss: 1.6155 

100%|██████████| 5/5 [02:17<00:00, 27.76s/it]
  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 2
 80%|████████  | 4/5 [01:54<00:26, 26.96s/it]Training loss: 1.5958 
100%|██████████| 5/5 [02:31<00:00, 30.03s/it]

Valid acc: 0.3438 

Training epoch 3
 80%|████████  | 4/5 [02:13<00:31, 31.62s/it]Training loss: 1.5827 
100%|██████████| 5/5 [02:37<00:00, 29.34s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 4
100%|██████████| 5/5 [02:39<00:00, 32.74s/it]
Training loss: 1.5740 

Valid acc: 0.3438 

Training epoch 5
100%|██████████| 5/5 [02:32<00:00, 31.54s/it]
Training loss: 1.5688 

Valid acc: 0.3438 
  0%|          | 0/5 [00:00<?, ?it/s]

Training epoch 6
 80%|████████  | 4/5 [01:55<00:29, 29.99s/it]Training loss: 1.5675 
100%|██████████| 5/5 [02:36<00:00, 33.48s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.1875 

Training epoch 7
100%|██████████| 5/5 [02:50<00:00, 31.76s/it]
Training loss: 1.5619 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.2188 

Training epoch 8
 80%|████████  | 4/5 [01:50<00:27, 27.23s/it]Training loss: 1.5573 
100%|██████████| 5/5 [02:27<00:00, 30.16s/it]

Valid acc: 0.3125 

Training epoch 9
100%|██████████| 5/5 [02:58<00:00, 34.73s/it]
Training loss: 1.5520 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.2188 

Training epoch 10
100%|██████████| 5/5 [03:01<00:00, 36.16s/it]
Training loss: 1.5460 

Valid acc: 0.3125 

Final Test acc: 0.2500 

*** Plotting validation accuracies and training losses *** (Done)

*** run number  2 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
  0%|          | 0/5 [00:00<?, ?it/s] (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
100%|██████████| 5/5 [02:47<00:00, 34.37s/it]
Training loss: 1.5967 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 2
 80%|████████  | 4/5 [02:11<00:30, 30.83s/it]Training loss: 1.5868 
100%|██████████| 5/5 [02:54<00:00, 34.61s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 3
 80%|████████  | 4/5 [02:32<00:35, 35.75s/it]Training loss: 1.5793 
100%|██████████| 5/5 [02:56<00:00, 32.27s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 4
100%|██████████| 5/5 [02:58<00:00, 36.67s/it]
Training loss: 1.5737 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 5
 80%|████████  | 4/5 [02:12<00:33, 33.25s/it]Training loss: 1.5710 
100%|██████████| 5/5 [02:56<00:00, 36.28s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3750 

Training epoch 6
100%|██████████| 5/5 [02:55<00:00, 37.50s/it]
Training loss: 1.5701 

Valid acc: 0.2188 
  0%|          | 0/5 [00:00<?, ?it/s]

Training epoch 7
 80%|████████  | 4/5 [02:42<00:39, 39.58s/it]Training loss: 1.5664 
100%|██████████| 5/5 [03:05<00:00, 34.81s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3750 

Training epoch 8
100%|██████████| 5/5 [02:41<00:00, 33.16s/it]
Training loss: 1.5657 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.2188 

Training epoch 9
100%|██████████| 5/5 [03:15<00:00, 37.91s/it]
Training loss: 1.5647 

Valid acc: 0.3125 
  0%|          | 0/5 [00:00<?, ?it/s]

Training epoch 10
100%|██████████| 5/5 [03:19<00:00, 40.52s/it]
Training loss: 1.5622 

Valid acc: 0.2812 

Final Test acc: 0.2917 

*** Plotting validation accuracies and training losses *** (Done)

*** run number  3 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
 (Done)
  "num_layers={}".format(dropout, num_layers))
*** Setting up adam optimizer *** (Done)
  0%|          | 0/5 [00:00<?, ?it/s]
# ##################### #
# Training han #
# ##################### #

Training epoch 1
 80%|████████  | 4/5 [02:20<00:36, 36.86s/it]Training loss: 1.5969 
100%|██████████| 5/5 [02:59<00:00, 37.42s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 2
 80%|████████  | 4/5 [02:18<00:32, 32.63s/it]Training loss: 1.5863 
100%|██████████| 5/5 [03:01<00:00, 35.82s/it]

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 3
100%|██████████| 5/5 [03:03<00:00, 34.02s/it]
Training loss: 1.5765 

Valid acc: 0.2500 

Training epoch 4
 80%|████████  | 4/5 [02:20<00:35, 35.26s/it]Training loss: 1.5704 

100%|██████████| 5/5 [03:03<00:00, 37.49s/it]
  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 5
100%|██████████| 5/5 [02:56<00:00, 36.73s/it]
Training loss: 1.5674 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3438 

Training epoch 6
100%|██████████| 5/5 [02:56<00:00, 37.75s/it]
Training loss: 1.5668 

Valid acc: 0.3125 
  0%|          | 0/5 [00:00<?, ?it/s]

Training epoch 7
100%|██████████| 5/5 [03:08<00:00, 35.21s/it]
Training loss: 1.5658 

  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.2188 

Training epoch 8
100%|██████████| 5/5 [02:45<00:00, 34.09s/it]
Training loss: 1.5623 

Valid acc: 0.1562 
  0%|          | 0/5 [00:00<?, ?it/s]

Training epoch 9
 80%|████████  | 4/5 [02:32<00:35, 35.85s/it]Training loss: 1.5587 

100%|██████████| 5/5 [03:16<00:00, 38.23s/it]
  0%|          | 0/5 [00:00<?, ?it/s]Valid acc: 0.3125 

Training epoch 10
100%|██████████| 5/5 [03:26<00:00, 41.10s/it]
Training loss: 1.5554 

Valid acc: 0.3125 

Final Test acc: 0.2083 

*** Plotting validation accuracies and training losses *** (Done)

Process finished with exit code 0
